### PR TITLE
Use subprocess timeout for SQL

### LINKED ISSUE #140: pysqlite causes segfaults on certain inputs
Hotfixing now by not importing the sqlite3 module. 

Filing this issue to figure out a permanent resolution

### PR DESCRIPTION
**Pull request by brianhou**
I haven't tested that this correctly terminates the sqlite subprocess when it takes too long.

This should be merged after #146. Resolves #140.

### CODE REVIEW

**Comment by albert12132** [comment]
Let me know when you've tested this.

**Comment by brianhou** [comment]
Actually tried it out and it seems to work. I added an example in the demo, so I think this is good to merge.

File: demo/doctest/hw1.py [hunk 1]
```diff
@@ -27,3 +27,11 @@ def double(x):
     """
     return x # Incorrect
 
+def forever():
```
Comment (albert12132) [inline demo/doctest/hw1.py]: This PR only affects `sqlite.py`, but this demo seems to imply it affects doctests as well. Is that the case?

Comment (brianhou) [inline demo/doctest/hw1.py]: Nope, I just added that to help me figure out the behavior for doctest.

**Comment by albert12132** [comment]
LGTM.

**Pull request merged**